src/include/catalog/settings_handle.h

```diff
@@ -102,4 +102,25 @@ class SettingsCatalogTable {
   catalog::SqlTableHelper *pg_settings_;
 };
 
+enum class SettingsTableColumn {
```

Comment:
Did these come from Peloton? I see most aren't used yet. I'm not clear on which ones we want yet, but would just like to understand the process that got them here.

Reply:
@pervazea is the author of `settings_handle`, we don't know how it works...

Reply:
The columns implemented in settiings_handle from Postgres pg_settings. Settings handle is "just a catalog table". You can set or read columns in the settings table.
Which ones are used is up to the team implementing settings.

The enum here though, seems unnecessary. The catalog code provides methods to access the columns. See jrolli's in progress catalog PR.